snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/components/SnomedRefSetMemberApiTest.java

```diff
@@ -800,8 +800,7 @@ public void executeInvalidAction() throws Exception {
 			"commitComment", "Executed invalid action on reference set member"
 		);
 
-		executeMemberAction(branchPath, memberId, invalidActionRequest).statusCode(400)
-		.body("message", CoreMatchers.equalTo("Invalid action type 'invalid'."));
+		executeMemberAction(branchPath, memberId, invalidActionRequest).statusCode(400);
```

Comment:
If there is a new error message in the response then please assert that.

Reply:
Done